setup.py

```diff
@@ -654,6 +655,11 @@ def preparations(cls):
     @classmethod
     def sundry_functions(cls) -> str:
         return super().sundry_functions() + '\n\n' + '''
+def get_blob_versioned_hashes(signed_blob_tx_rlp: bytes) -> Sequence[VersionedHash]:
+    decoded_tx = rlp.decode(signed_blob_tx_rlp, SignedBlobTransaction)
+    return [VersionedHash(x) for x in decoded_tx.blob_versioned_hashes]
```

Comment:
Would it be possible to use a "neutral" encoding here, that does not add RLP to the CL? It seems like the wrong direction to go with.